### DISCUSSION TITLE
Would The Scoppy W work with Pi Pico + ESP8266

### DISCUSSION THREAD

**Discussion by Lanson2232** [fhdm-dev/scoppy#82] [Q&A]
Could the Scoppy W build work on pi pico + ESP8266 if yes how? what do I need to follow or does a Desktop Client exists for scoppy because my phone is USB-C I didn't find any OTG on stock and would prefer do it over the WIFI or it would be awesome for giving out details what kind of data is transferring to help out creating more "Client" for such project

**Comment by fhdm-dev** [comment]
Hi,

Sorry but it definitely won't work with  a Pico + ESP8266.

**Comment by dduran8** [comment]
Is there any chance to make it work with any other wifi module other than ESP8266 ?

**Reply by fhdm-dev** [reply]
Hi,
Sorry but I don't have any plans for it to work with other Wi-Fi modules. The Wi-Fi connectivity will only work with the Pico W.

**Comment by ExperiMentor** [comment]
I didn't have any trouble finding a USB-C to microUSB cable which works fine. Because USB-C in effect has OTG built-in, it does not look like an OTG adaptor. For example, this, though mine is longer [https://www.ebay.co.uk/itm/195419598892](url)
